Vienna.xcodeproj/project.pbxproj

```diff
@@ -7112,6 +7112,7 @@
 				CLANG_WARN_UNREACHABLE_CODE = YES;
 				CLANG_WARN__DUPLICATE_METHOD_MATCH = YES;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
+				GCC_NO_COMMON_BLOCKS = YES;
```

Comment:
These changes should be in the `configs/Project-All.xcconfig` file rather than the project file itself.